src/main/java/com/odysseusinc/arachne/portal/service/impl/BaseUserServiceImpl.java

```diff
@@ -624,12 +621,14 @@ public void saveAvatar(U user, MultipartFile file)
         if (!userFilesDir.exists()) {
             userFilesDir.mkdirs();
         }
-        final File avatar = Paths.get(userFilesDir.getPath(), "avatar.jpg").toFile();
+        final File avatar = Paths.get(userFilesDir.getPath(), AVATAR_FILE_NAME).toFile();
```

Comment:
How is it going to work with all stuff below? Detection of orientation, image preparation, etc. E.g. there is JpegDirectory class later